VideoLocker/res/layout/activity_register.xml

```diff
@@ -91,13 +91,10 @@
                     style="@style/semibold_text"
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:layout_marginLeft="12dp"
-                    android:layout_marginStart="12dp"
-                    android:layout_marginRight="12dp"
-                    android:layout_marginEnd="12dp"
+                    android:layout_margin="@dimen/edx_x_small"
                     android:focusable="true"
                     android:text="@string/show_optional_text"
-                    android:textSize="14sp"
+                    android:textSize="@dimen/edx_small"
                     tools:targetApi="17" />
```

Comment:
If we plan to not have `marginEnd` and `marginStart` attributes, then this attribute can be deleted.